src/js/view/controls/settings-menu.js

```diff
@@ -85,13 +97,24 @@ export function setupSubmenuListeners(settingsMenu, controlbar, viewModel, api)
             return;
         }
 
+        const currentLevel = model.get('currentLevel');
+
         addQualitiesSubmenu(
             settingsMenu,
             levels,
             (index) => api.setCurrentQuality(index),
-            model.get('currentLevel'),
+            currentLevel,
             model.get('localization').hd
         );
+
+        if (currentLevel === 0) {
+            // Set quality label again after preroll on HLS on Android Chrome
+            changeAutoLabel(
+                model.get('mediaModel').get('visualQuality'), 
```

Comment:
I see us getting visualQuality from the model `model.get('visualQuality')`, not the mediaMedia elsewhere in this file.